rust/experimental/query_engine/kql-parser/src/kql.pest

```diff
@@ -183,9 +183,11 @@ summarize_expression = {
 	"summarize"
     	~ (aggregate_assignment_expression ~ ("," ~ aggregate_assignment_expression)*)?
         ~ ("by" ~ group_by_expression ~ ("," ~ group_by_expression)*)?
-        ~ ("|" ~ tabular_expressions)*
+        ~ ("|" ~ _tabular_whitespace ~ tabular_expressions)*
```

Comment:
Up at the top there is a `WHITESPACE` rule. If you add tab into that does it take care of it across the board? Probably OK to treat tab in the same way as space everywhere.